src/react/components/searchResults.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import sanitize from "sanitize-html";
+import HeroIcon from './heroIcon'
+
+const SearchResults = ({ results }) => {
+  if (!results.hits) {
+    return <div />;
+  }
+
+  return (
+    <div id="search-results" className="su-sans">
+      <div className="algolia-search--results-count lg:mb-16">
+        <span className="font-semibold">{results.nbHits}</span> results:
+      </div>
+      {results.hits.map((result) => (
+        <div
+          key={result.objectID}
```

Comment:
I might be remembering this wrong but I think we may have found that objectID could contain a space, but that might have been from Storyblok and not Algolia.